core/src/main/java/org/mskcc/cbio/portal/util/CnaUtil.java

```diff
@@ -70,9 +80,18 @@ public CnaEvent createEvent(GeneticProfile geneticProfile, int sampleId, String[
         cna.setDriverFilterAnnotation(TabDelimitedFileUtil.getPartString(getColumnIndex(CnaUtil.CBP_DRIVER_ANNOTATION), parts));
         cna.setDriverTiersFilter(TabDelimitedFileUtil.getPartString(getColumnIndex(CnaUtil.CBP_DRIVER_TIERS), parts));
         cna.setDriverTiersFilterAnnotation(TabDelimitedFileUtil.getPartString(getColumnIndex(CnaUtil.CBP_DRIVER_TIERS_ANNOTATION), parts));
+        cna.setAnnotationJson(
+            convertMapToJsonString(
+                this.namespaceColumnParser.parseCustomNamespaces(parts)
+            )
+        );
         return cna;
     }
-
```

Comment:
```java
    public int getColumnIndex(String colName) {
        Integer index = this.columnIndexMap.get(colName.toLowerCase());
        if (index == null) {
            index = -1;
        }
        return index;
    }
```

This entire function is annoying me. Can we at least make it private? It's only used internally. Bonus points if you switch the `get` for `getOrDefault`

Reply:
Fixed in [7bccc41](https://github.com/cBioPortal/cbioportal/compare/9ce311d9b4a4cabb7012450c85e285ba553fa5af..7bccc41e1e6dd57d97f8e258889a730f8fb019a7)